CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ##### Breaking
 
-* None.
+* Fixed: SwiftLint assumes paths in the YAML config file are relative to the
+current directory even when `--path` is passed as an argument.  
```

Comment:
elsewhere in this doc, multi-line list items are aligned with the first line (i.e. this needs to spaces)